docs/guide/pt-BR/README.md

```diff
@@ -0,0 +1,117 @@
+# Yii Injector
+
+## Uso geral
+
+```php
+use Yiisoft\Di\Container;
+use Yiisoft\Di\ContainerConfig;
+use Yiisoft\Injector\Injector;
+
+$config = ContainerConfig::create()
+    ->withDefinitions([
+        EngineInterface::class => EngineMarkTwo::class,
+    ]);
+$container = new Container($config);
+
+$getEngineName = static function (EngineInterface $engine) {
+    return $engine->getName();
+};
+
+$injector = new Injector($container);
+echo $injector->invoke($getEngineName);
+// outputs "Mark Two"
```

Comment:
Should be translated.

Reply:
All codes are kept in the original, in English. Could you clarify the doubt?

Reply:
That's not code, that's comment in the code explaining it. Usually we're translating these.

Reply:
I understood. I will review all the packages

Reply:
done